### ISSUE TITLE
Under Journal, we need to make the articles navigable through a keyboard

### ISSUE THREAD

**Issue by maxcell** [netlify-templates/gatsby-ecommerce-theme#7]
## Summary

As a keyboard user, I should be able to see the card and use the `TAB` and `SHIFT+TAB` combination to focus along each as a link
![image](https://user-images.githubusercontent.com/8431042/168886882-ba8377ce-2c98-48a7-8f2d-22a9a9e4168a.png)
